pilot/helpers/agents/TechnicalWriter.py

```diff
@@ -42,11 +42,7 @@ def create_readme(self):
             "files": self.project.get_all_coded_files(),
         }, GET_DOCUMENTATION_FILE)
 
-        changes = self.project.developer.replace_old_code_comments([llm_response])
-
-        for file_data in changes:
-            self.project.save_file(file_data)
-
+        self.project.save_file(llm_response)
```

Comment:
i believe that with these new changes this has to go through codeMonkey and not be saved directly but double check.

Reply:
This saves file correctly (as in - what we get from GPT gets saved correctly).

Though in theory that could have a problem of LLM not sending a complete file, since there's no other constraints and we always ask for full rewrite, I don't think we need to involve CodeMonkey here.